package.json

```diff
@@ -18,15 +18,14 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@playwright/test": "^1.45.0",
+    "@playwright/test": "^1.49.0",
     "@types/node": "^20.11.17",
-    "browserstack-node-sdk": "^1.34.18",
+    "browserstack-node-sdk": "^1.34.23",
```

Comment:
QUESTION: I saw you removed references to browserstack-node-sdk from the command-line above. Is the package still needed?

Reply:
yes, it's still needed in package.json line 15